app/uk/gov/hmrc/tai/connectors/TaxCodeChangeConnector.scala

```diff
@@ -41,18 +41,21 @@ class TaxCodeChangeConnector @Inject()(
 
   override val originatorId = config.originatorId
 
-  implicit private val header: HeaderCarrier = {
+  implicit private def header(implicit hc: HeaderCarrier): HeaderCarrier = {
```

Comment:
can we rename this to something more meaningful like `createHeader`

Reply:
Done , changed name to createHeader